shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -633,6 +634,16 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
       object.accessibilityCustomActions = accessibilityCustomActions;
     }
 
+    if (node.HasFlag(flutter::SemanticsFlags::kIsLiveRegion)) {
```

Comment:
Should we remove the mapping of kIsLiveRegion to UpdatesFrequently? I believe giving it that trait doesn't really make sense for us as we currently always fire layoutchanged notifications?

Reply:
Yes, removed